storage/storage_model.go

```diff
@@ -103,3 +119,55 @@ func (m *buildMetadata) toJSON() ([]byte, error) {
 
 	return metadataJSON, nil
 }
+
+func (m *buildMetadata) fromJSON(data []byte) error {
+	return errors.Wrap(json.Unmarshal(data, m), "unmarshalling metadata")
+}
+
+type testMetadata struct {
+	ID      string `json:"id"`
+	Name    string `json:"name"`
+	BuildID string `json:"build_id"`
+	TaskID  string `json:"task_id"`
+}
```

Comment:
Just to make sure: we're not holding on to the phase and command (from [here](https://github.com/evergreen-ci/logkeeper/blob/3de44c77ad289efb1e04246e61e03a3da37e5ef3/views.go#L171-L172)) because we never use them?

Reply:
Yeah, I don't think we expose it anywhere? We don't serialize the json of our tests anywhere AFAICT?

Reply:
I don't see it anywhere either.
We use name [here](https://github.com/evergreen-ci/logkeeper/blob/0bd05c4d927524c21ee7cc48d2e87767b783b644/views.go#L522), but I don't see anything else.